shared/app/nav.desktop.js

```diff
@@ -5,26 +5,58 @@ import * as RouteTree from '../route-tree/render-route'
 import GlobalError from './global-errors/container'
 import Offline from '../offline/container'
 import TabBar from './tab-bar/container'
+import {isDarwin} from '../constants/platform'
 import {Box, ErrorBoundary} from '../common-adapters'
-import {chatTab, loginTab, type Tab} from '../constants/tabs'
+import {
+  chatTab,
+  fsTab,
+  loginTab,
+  peopleTab,
+  teamsTab,
+  devicesTab,
+  gitTab,
+  settingsTab,
+  walletsTab,
+  type Tab,
+} from '../constants/tabs'
+import {navigateTo} from '../actions/route-tree'
 import {connect, type TypedState, type Dispatch} from '../util/container'
 import {globalStyles} from '../styles'
 import RpcStats from './rpc-stats'
 
 type Props = {
   layerScreens: I.Stack<RouteTree.RenderRouteResult>,
+  onHotkey: (cmd: string) => void,
   visibleScreen: RouteTree.RenderRouteResult,
   routeSelected: Tab,
   routePath: I.List<string>,
 }
 
+const hotKeyTabMap: {[string]: Tab} = {
+  '1': peopleTab,
+  '2': chatTab,
+  '3': fsTab,
+  '4': teamsTab,
+  '5': devicesTab,
+  '6': gitTab,
+  '7': settingsTab,
+  '8': walletsTab,
+}
+
 class Nav extends React.Component<Props> {
   render() {
     const {routeSelected, routePath, visibleScreen, layerScreens} = this.props
     return (
       <ErrorBoundary>
         <Box style={stylesTabsContainer}>
-          {routeSelected !== loginTab && <TabBar routeSelected={routeSelected} routePath={routePath} />}
+          {routeSelected !== loginTab && (
+            <TabBar
+              hotkeys={Object.keys(hotKeyTabMap).map(key => `${isDarwin ? 'command' : 'control'}+${key}`)}
```

Comment:
just make this once in this module, not every time this renders or it'll just keep rerendering as it changes